processor/intervalprocessor/go.mod

```
@@ -0,0 +1,71 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/processor/intervalprocessor
+
+go 1.20
+
+require (
+	github.com/open-telemetry/opentelemetry-collector-contrib/internal/coreinternal v0.92.0
+	github.com/stretchr/testify v1.8.4
+	go.opentelemetry.io/collector/component v0.93.1-0.20240124123350-9047c0e373f9
+	go.opentelemetry.io/collector/confmap v0.93.1-0.20240124123350-9047c0e373f9
+	go.opentelemetry.io/collector/consumer v0.93.1-0.20240124123350-9047c0e373f9
+	go.opentelemetry.io/collector/pdata v1.0.2-0.20240124123350-9047c0e373f9
+	go.opentelemetry.io/collector/processor v0.93.0
```

Unfortunately a check is failing due to this.
```suggestion
	go.opentelemetry.io/collector/processor v0.93.1-0.20240124123350-9047c0e373f9
```